config/samples/keystone_v1beta1_keystoneservice.yaml

```diff
@@ -3,5 +3,18 @@ kind: KeystoneService
 metadata:
   name: keystoneservice-sample
 spec:
-  # Add fields here
-  foo: bar
+  username: admin
+  password: foobar123
```

Comment:
Should we have this input for 'password' use a k8s secret?

Reply:
Note the CI failures:
https://travis-ci.com/github/openstack-k8s-operators/keystone-operator/jobs/384834414